go/plugins/internal/gemini/gemini.go

```diff
@@ -265,7 +276,10 @@ func convertRequest(client *genai.Client, model string, input *ai.ModelRequest)
 	choice := convertToolChoice(input.ToolChoice, input.Tools)
 	gc.ToolConfig = choice
 
-	// TODO: configure cache
+	// configure cache
```

Comment:
Nit: this and other comments like it aren't that helpful in my opinion. Comments should describe why not what. We can see the what from the code. It's just visual noise and easy for them to become outdated. I would suggest removing them throughout these files.